src/storage-folder.coffee

```diff
@@ -0,0 +1,27 @@
+path = require "path"
+fs = require "fs-plus"
+
+module.exports =
+class StorageFolder
+  constructor: (containingPath) ->
+    @path = path.join(containingPath, "storage")
+
+  store: (name, object) ->
+    fs.writeFileSync(@pathForKey(name), JSON.stringify(object), 'utf8')
```

Comment:
> So I don't have a strong opinion about it, but generally I err on the side of not swallowing errors. Does this case warrant it?

To me this feels like the same case to wrap the `JSON.parse` in a try/catch down on line 22, better safe than sorry.